modelina-website/README.md

```diff
@@ -16,4 +16,39 @@ Here is a quick overview of where some of the functions for rendering the playgr
 - `src/components/playground/PlaygroundOptions.tsx` is the main component that renders the options based on which output is selected.
 - `src/components/playground/options` contain all the individual react components for showing the output options.
 - `src/components/playground/Playground.tsx` is the main playground component, and is the one rendered by the playground page.
-- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
\ No newline at end of file
+- `src/components/playground/GeneratedModels.tsx` is the playground component responsible for rendering the generated models.
+
+## Contributors flow
+
+1. Fork the repository by clicking on `Fork` option on top right of the main repository.
+
+2. Open Command Prompt on your local computer.
+
+3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
+   For multiple contributions it is recommended to have [proper configuration of forked repo](https://github.com/asyncapi/community/blob/master/git-workflow.md).
+
+```bash
+    git clone https://github.com/<username>/website/
+```
```

Comment:
I think we should just link to https://github.com/asyncapi/community/blob/master/git-workflow.md and that's it, at least for the first 3 points 🙂 

No need to repeat what it say.

Reply:
Ohhhh, I get you @jonaslagoni. Let's do that indeed no need to repeat we shall just link to https://github.com/asyncapi/community/blob/master/git-workflow.md